package.json

```diff
@@ -8,9 +8,9 @@
     "url": "https://github.com/mooyoul/proxyfront.git"
   },
   "scripts": {
-    "build": "check-engine && rm -Rf dst && tsc && cd src && find . \\( -name '*.json' -o -name '*.html' \\) -type f -exec cp {} ../dst/{} \\;  && cd ..",
-    "pack": "rm -f dst.zip && cp package.json dst/package.json && (cp package-lock.json dst/ || true) && (cp .npmrc dst/.npmrc || true) && cd dst && npm install --cache=../.npm --production && npm ls && zip -rqy ../dst.zip . && cd ..",
-    "pretest": "check-engine && rm -Rf dst && tsc -p ./tsconfig.test.json && cd src && find . -name '*.json' -type f -exec cp {} ../dst/{} \\; && cd ..",
+    "build": "rm -Rf dst && tsc && cd src && find . \\( -name '*.json' -o -name '*.html' \\) -type f -exec cp {} ../dst/{} \\;  && cd ..",
+    "pack": "rm -f dst.zip && cp package.json dst/package.json && (cp package-lock.json dst/ || true) && (cp .npmrc dst/.npmrc || true) && cd dst && npm install --cache=../.npm && npm ls && zip -rqy ../dst.zip . && cd ..",
```

Comment:
Removing `--production` flag results to lager deployment package size. It will include entire dependencies including devDep.

Reply:
Ah thanks. I re-added the flag and also added it to the `npm ls` call so it's not failing the command chain anymore